src/control/lib/control/pool_property.go

```diff
@@ -116,6 +123,50 @@ func PoolProperties() PoolPropertyMap {
 				jsonNumeric: true,
 			},
 		},
+		"ec_pda": {
+			Property: PoolProperty{
+				Number:      drpc.PoolPropertyECPda,
+				Description: "Performance domain affinity level of EC",
+				valueHandler: func(s string) (*PoolPropertyValue, error) {
+					ecpdaErr := errors.Errorf("invalid ec_pda value %s", s)
```

Comment:
minor: It's usually better to use the `%q` directive for showing the string input as it will automatically quote it, which makes it easier to see when the user has somehow supplied an empty string as the argument.